include/sundials/sundials_iterative.h

```diff
@@ -78,7 +78,7 @@ enum
  * -----------------------------------------------------------------
  */
 
-enum
+enum SUN_GRAMSCHMIDT_ID
```

Comment:
To follow the `enum` naming convention, should this be `SUNGramSchmidtID` or maybe `SUNGramSchmidtType`? I like `Type` better than `ID`.
```suggestion
enum SUNGramSchmidtType
```